models/intel/resnet101-int8-tf-0001/description/resnet101-int8-tf-0001.md

```diff
@@ -0,0 +1,54 @@
+# resnet101-int8-tf-0001
+
+## Use Case and High-Level Description
+
+This is the Resnet-101 v1 model that is designed to perform image classification. 
+The model has been pretrained on the ImageNet image database and then quantized to INT8 fixed-point precision using 
+so-called Quantization-aware training approach implemented in TensorFlow framework. 
+For details about the original floating point model, check out the [paper](https://arxiv.org/pdf/1512.03385.pdf).
+
+The model input is a blob that consists of a single image of "1x224x224x3" in BGR order.
+
+The model output for `resnet101-int8-tf-0001` is the usual object classifier output for the 1000 different classifications matching those in the ImageNet database.
+
+## Example
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Classification|
+| GFLOPs            | 14.441         |
+| MParams           | 44.496        |
+| Source framework  | TensorFlow    |
+
+## Accuracy
+
+The quality metrics calculated on ImageNet validation dataset is 74.5% accuracy top-1.
```

Comment:
Duplication of table content, please remove (except, reference to dataset)

Reply:
@eizamaliev this is consistent description with already released quantized/sparsified models inceptionv3-int8-xxx, mobilenetv2-int8-xxx, resnet-50-int8-xxx.